.gitignore

```diff
@@ -146,3 +146,9 @@ dist
 
 # Locally generated PII reports
 pii_report
+
+# transifex configurations
+.transifexrc
```

Comment:
Can we add this file into git but pull the API key from the OS environment or `lms.yml/cms.yml`?